modules/virt-infer-instancetype-preference.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-specifying-instance-preference.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-infer-instancetype-preference_{context}"]
+= Inferring an instance type or preference
+
+You can apply labels to the boot volume to infer which instance type or preference to use to define the runtime characteristics of a VM by using the `--infer-instancetype` and `--infer-preference` flags.
+
+If no flags are applied, the `inferFromVolumeFairlure` policy of the `inferFromVolume` attribute defaults to `Ignore`. Any errors when inferring from the boot volume are ignored, and the VM is created with instancetype and preference left unset.
+
+However, when the flags are applied, the `inferFromVolumeFailure` policy is set to `Reject`. Any errors when inferring from the boot volume result in the rejection of the creation of that VM.
+
+.Prerequisites
```

Comment:
@0xFelix Do we have any prereqs for this?

Reply:
Nothing special I would say, `virtctl` needs to be available.